samples/geography/insert_geojson.py

```diff
@@ -12,14 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import Dict, Mapping, Optional, Sequence
 
-def insert_geojson(override_values={}):
+
+def insert_geojson(
+    override_values: Optional[Mapping[str, str]] = None
+) -> Sequence[Dict[str, object]]:
     # [START bigquery_insert_geojson]
     import geojson
     from google.cloud import bigquery
 
     bigquery_client = bigquery.Client()
 
+    if override_values is None:
+        override_values = {}
+
```

Comment:
Let's move the `override_values` outside of the actual sample part.

```suggestion
    if override_values is None:
        override_values = {}

    # [START bigquery_insert_geojson]
    import geojson
    from google.cloud import bigquery

    bigquery_client = bigquery.Client()

```

Alternatively, we can put it after `# [END bigquery_insert_geojson]`, but GitHub won't let me suggest modifications including line 33.

Alternatively again, we could switch to the `table_id_original` pattern we've started using in some other samples.

Reply:
+1

Reply:
I can't find any references to `table_id_original`?

I will thus just move the `is None` check outside of the sample.